pythainlp/corpus/core.py

```diff
@@ -70,7 +70,10 @@ def path_pythainlp_corpus(filename: str) -> str:
     return os.path.join(corpus_path(), filename)
 
 
-def get_corpus(filename: str, as_is: bool = False) -> Union[frozenset, list]:
+def get_corpus(filename: str,
+               as_is: bool = False,
+               comments: bool = True
```

Comment:
I have changed this to `comments` instead of `discard_comments` (as I suggested earlier) to avoid double negation. 

The semantic now is:
- if `comments` = True, then keep comments
- if `comments` = False, then discard comments